src/app/GitUI/TranslatedStrings.cs

```diff
@@ -336,6 +337,10 @@ public static void Reinitialize()
         private readonly TranslationString _scriptText = new("Script");
         private readonly TranslationString _scriptUserInputCaption = new("User input for script '{0}'");
         private readonly TranslationString _scriptUserCanceledRun = new("Script not started.");
+        public static string Favorites
+        {
+            get => _instance.Value._favoriteText.Text;
+        }
```

Comment:
To be moved behind `#endregion` and to be adapted to the surrounding code style.